app/build.gradle

```diff
@@ -69,6 +69,9 @@ dependencies {
     implementation "org.jetbrains.kotlinx:kotlinx-coroutines-android:1.5.0"
     implementation "org.jetbrains.kotlinx:kotlinx-coroutines-core:1.5.0"
 
+    // Android auto module
+//    implementation(project(":android-auto"))
+
```

Comment:
Remove this?

Reply:
Yep, I tried this over here just to see the issues. Better to have a `android-auto-app`